scripts/parser-tests/test262/allowlist.txt

```diff
@@ -6,6 +6,9 @@ language/import/import-attributes/json-named-bindings.js(default)
 language/import/import-attributes/json-named-bindings.js(strict mode)
 
 
-# 0 valid programs produced a parsing error
-
+# 4 valid programs produced a parsing error
 
+built-ins/DisposableStack/prototype/Symbol.dispose.js(default)
+built-ins/DisposableStack/prototype/Symbol.dispose.js(strict mode)
+built-ins/DisposableStack/prototype/disposed/getter.js(default)
+built-ins/DisposableStack/prototype/disposed/getter.js(strict mode)
```

Comment:
The parsing errors come from upstream: https://github.com/tc39/test262/pull/4243. It will be resolved in the next scheduled update.